server/server.go

```diff
@@ -0,0 +1,43 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	changelogs "github.com/crossplane/crossplane-runtime/apis/changelogs/proto/v1alpha1"
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+)
+
+type Server struct {
```

Comment:
Should this have godoc? Looks like this repo has no linter.

Reply:
yeah that's a good point - it looks like CI linting didn't run at all, which maybe is because the CI files are brand new in this PR...i'll manually lint first then ensure linting is running on subsequent PRs..

Reply:
ah, we were missing a `.golangci.yml` config file here, that started finding all the linter issues now when running locally. Fixing all those up!